<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.00568182 rgba(1, 142, 190, 255), stop:1 rgba(255, 255, 255, 255))
}
</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>571</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color:rgb(54, 54, 81);
	font: 63 12pt &quot;Verdana Pro Semibold&quot;;
	background-color:transparent;
}</string>
   </property>
   <property name="text">
    <string>*Before begin,please login into your mail account to manage them</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color:rgb(54, 54, 81);
	font: 63 30pt &quot;Verdana Pro Semibold&quot;;
	background-color:transparent;
}</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color:rgb(54, 54, 81);
	font: 63 10pt &quot;Verdana Pro Semibold&quot;;
	background-color:transparent;
}</string>
   </property>
   <property name="text">
    <string>*email</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>300</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color:rgb(54, 54, 81);
	font: 63 10pt &quot;Verdana Pro Semibold&quot;;
	background-color:transparent;
}</string>
   </property>
   <property name="text">
    <string>*password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="email">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	color:rgb(54, 54, 81);
	font: 63 14pt &quot;Verdana Pro Semibold&quot;;
border-radius:8px;
background-color:transparent;
border:1px solid  rgb(54, 54, 81);
padding:2px
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>320</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	color:rgb(54, 54, 81);
	font: 63 14pt &quot;Verdana Pro Semibold&quot;;
border-radius:8px;
background-color:transparent;
border:1px solid  rgb(54, 54, 81);
padding:2px
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="login_btn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>380</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
color:rgb(0, 255, 0);
font: 63 10pt &quot;Verdana Pro Semibold&quot;;
background-color:rgb(54, 54, 81);
border-radius:8px;
}
QPushButton:hover{
	background-color:rgb(41, 41, 61);
}
</string>
   </property>
   <property name="text">
    <string>login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close_btn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>380</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
color:#fff;
font: 63 10pt &quot;Verdana Pro Semibold&quot;;
background-color:rgb(255, 66, 66);
border-radius:8px;
}
QPushButton:hover{
	background-color:rgb(255, 20, 20);
}
</string>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="error">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>561</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color:rgb(255, 0, 0);
	font: 63 10pt &quot;Verdana Pro Semibold&quot;;
	background-color:transparent;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
